checkov/secrets/runner.py

```diff
@@ -231,7 +232,7 @@ def run(
                 self.pbar.close()
 
             secret_records: dict[str, SecretsRecord] = {}
-            secrets_in_uuid_form = ['CKV_SECRET_116', 'CKV_SECRET_30']
+            secrets_in_uuid_form = ['CKV_SECRET_116', 'CKV_SECRET_48', 'CKV_SECRET_40', 'CKV_SECRET_30']
```

Comment:
Why adding 2?

Reply:
these are for another ticket in secrets, i just put them in the same PR